App.js

```diff
@@ -1,25 +1,66 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * @format
- * @flow
- */
-
-import React from 'react';
-import {StyleSheet, View, Text} from 'react-native';
-
+import React, {useState, useEffect} from 'react';
+import {StyleSheet, View, ActivityIndicator} from 'react-native';
 import {Colors} from 'react-native/Libraries/NewAppScreen';
+import Items from './components/Items/Items';
+import Buttons from './components/Buttons/Buttons';
 
 const App = () => {
+  const [isLoading, setLoading] = useState(true);
+  const [sorts, setSorts] = useState({id: false, author: false});
```

Comment:
It will be better to use just `setData` hook to modify arrays. 
Sorting during rendering is a performance issue.